data/remnant/remnant jobs.txt

```diff
@@ -95,6 +95,7 @@ mission "Remnant: Broken Jump Drive job"
 	destination
 		government "Remnant"
 		attributes outfitter
+		not attributes "remnant station"
```

Comment:
Should this one also be filtering out places that require gaslining?
```suggestion
		not attributes "requires: gaslining"
		not attributes "remnant station"
```

Reply:
I'd say the second option.

Reply:
Good eye, but no. Not all of the research is deemed likely to cause cataclysms, so some of it could potentially take place elsewhere. Just the restriction on the station is sufficient.